R/check_class-friendly_class.R

```diff
@@ -0,0 +1,148 @@
+friendly_class <- function(object) {
```

Comment:
I think this has to be exported for another external package to use the generic. I tried testing importing it from another package and got this:

```
Error: object ‘friendly_class’ is not exported by 'namespace:tblcheck'
```

```suggestion
#' @export
friendly_class <- function(object) {
```